### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#189] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-20 223950](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/c93bb4c6-ef30-439d-8c12-76248993c458)

**Comment by cornflexxx** [comment, marked as answer]
Possiamo interpretare la richiesta del problema nel seguente modo:
quanti lanci dobbiamo fare per stimare il parametro $p$ con un errore inferiore a $\delta$ nel 95% dei casi? Sappiamo per la legge dei grandi numeri che quando il numero di lanci $n\to\infty$, allora la probabilità di approssimare $p$ tramite il rapporto di teste sul numero totale di lanci ($\frac{S_n}{n}$) con un errore inferiore a $\delta$ è certa. In questo caso però ci stiamo accontentando di un valore di $n$ più piccolo che però  ci garantisca  una probabilità del 95%. Quindi dobbiamo trovare un $N\in \mathbb{N}$, per cui quando $n\ge N$ allora:
$$\mathbb{P} [|\frac{S_n}{n}-p|\lt \delta]\ge 0.95\\quad (*)$$

Essendo la situazione del lancio di una moneta proprio il caso di una distribuzione binomiale , considerando $S_n$ la $v.a$ che conta il numero di teste in $n$ lanci, allora il suo valore atteso è proprio $np$, e quindi:
$$\mathbb{E}(S_n/n)=\frac{1}{n}\mathbb{E}(S_n)=p$$

Dopo aver fatto questa osservazione riscrivo la prima probabilità con la seguente forma, equivalente alla precedente:

$$\mathbb{P} [|\frac{S_n}{n}-\mathbb{E}(S_n/n)|\geq \delta]\le 0.05$$

Ma questa è proprio la forma della **disuguaglianza di Chebyshev**:

$$\mathbb{P} [|\frac{S_n}{n}-\mathbb{E}(S_n/n)|\geq \delta]\le \frac{\mathbb{V}(S_n/n)}{\delta ^2}$$

Ne segue che:
$$\frac{\mathbb{V}(S_n/n)}{\delta ^2}\le 0.05$$

Troviamo la varianza di $S_n/n$:

$$\mathbb{V}(S_n/n)=\frac{1}{n^2}\cdot \mathbb{E}[(S_n-\mathbb{E}[S_n])^2]=\frac{1}{n^2}\cdot np(1-p)=\frac{p(1-p)}{n}$$

Allora abbiamo trovato $N$:
$$N=\frac{p(1-p)}{0.05 \cdot \delta ^2}$$

Affinche la $(*)$ si realizzi dobbiamo porre $n\ge N$.

 $n\ge N$ quando $n\ge \frac{max(p(1-p))}{\delta^2 \cdot 0.05}$, quindi la $f(p)=p-p^2$ è massima quando $p=1/2$ quindi $f(1/2) = 1/4$ $\longrightarrow n\ge\frac{1}{4\delta^2\cdot 0.05}$

**Comment by CiottoloMaggico** [comment]
Risultato analogo a @cornflexxx 
L'unica differenza è che per comodità ho riscritto 0.05 come 1/20
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/3abab252-eeff-4cf9-aa42-3546974f7f8d)
